ivy/functional/frontends/torch/nn/functional/loss_functions.py

```diff
@@ -532,3 +532,35 @@ def multilabel_soft_margin_loss(
     ret = reduction(loss)
 
     return ret
+
+
+
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes({"1.13.0 and below": ("float16",)}, "torch")
+
+def ctc_loss(
+    log_probs,
+    targets,
+    input_lengths,
+    target_lengths,
+    blank = 0,
+    reduction = "mean",
+    zero_infinity = False,
+):
+
+    log_probs = ivy.swapaxes(log_probs, 0, 1)
+    labels = ivy.astype(targets, "int32")
+    input_length = ivy.reshape(input_lengths, [-1])
+    label_length = ivy.reshape(target_lengths, [-1])
+
+    try:
+        ret = ivy.ctc_loss(log_probs, labels, input_length, label_length, blank=blank, zero_infinity=zero_infinity)
+
+    except ValueError as e:
+        raise Ivy
+
+
+    ret = _apply_reduction(reduction, ret)
+    return ret
```

Comment:
no need for the `try except`
```suggestion

    ret = ivy.ctc_loss(log_probs, labels, input_length, label_length, blank=blank, zero_infinity=zero_infinity)
    ret = _apply_reduction(reduction, ret)
    return ret
```